lib/src/generator/template_data.dart

```diff
@@ -254,13 +252,13 @@ abstract class InheritingContainerTemplateData<T extends InheritingContainer>
   @override
   String get htmlBase => '../';
 
-  Class get objectType {
+  Class? get objectType {
     if (_objectType != null) {
-      return _objectType;
+      return _objectType!;
     }
 
     var dc = _packageGraph.libraries
-        .firstWhere((it) => it.name == 'dart:core', orElse: () => null);
+        .firstWhereOrNull((it) => it.name == 'dart:core');
 
     return _objectType = dc?.getClassByName('Object');
```

Comment:
whoa, there is a way faster way to do this now (using Specials), but outside the scope of this PR.

Reply:
Haha oh?

Reply:
yep.  _packageGraph.specialClasses[SpecialClass.object] should get you the right `Object`.